pythran/transformations/remove_fstrings.py

```diff
@@ -0,0 +1,48 @@
+"""Turns f-strings to format syntax with modulus"""
+
+import gast as ast
+
+from pythran.passmanager import Transformation
+from pythran.syntax import PythranSyntaxError
+
+
+class RemoveFStrings(Transformation, ast.NodeTransformer):
+    """Turns f-strings to format syntax with modulus
+
+    >>> import gast as ast
+    >>> from pythran import passmanager, backend
+    >>> node = ast.parse("f'a = {1+1:4d}; b = {b:s};'")
+    >>> pm = passmanager.PassManager("test")
+    >>> _, node = pm.apply(RemoveFStrings, node)
+    >>> print(pm.dump(backend.Python, node))
+    ('a = %4d; b = %s;' % ((1 + 1), b))
+    """
+
+    def visit_JoinedStr(self, node):
+        if not any(
+            isinstance(value, ast.FormattedValue) for value in node.values
+        ):
+            # nothing to do (not a f-string)
+            return node
+        base_str = ""
+        elements = []
+        for value in node.values:
+            if isinstance(value, ast.Constant):
+                base_str += value.value.replace("%", "%%")
+            elif isinstance(value, ast.FormattedValue):
+                base_str += "%"
+                if value.format_spec is None:
+                    raise PythranSyntaxError(
+                        "Oops sorry, f-strings without format specifier are "
+                        f"not supported (l. {value.lineno})."
```

Comment:
I like the pun of using a fstring to state that this fstring is not supported ;-)

Reply:
Pythran synatx error take a few extra parameter that helps spotting the location instead of handling that on you own (basically passing `value` as a second parameter should work.)